docs/src/algorithms/sampling.md

```diff
@@ -65,6 +65,21 @@ blocks  = sample(grid, sampler)
 viz(blocks)
 ```
 
+### FibonacciSampling
+```@docs
+FibonacciSampling
+```
+
+```@example sampling
+sphere = Sphere((0.,0.,0.), 1.)
+
+# sample points using the Fibonacci lattice method
+sampler  = FibonacciSampling(100)
+points  = sample(sphere, sampler) |> collect
+
+viz(points)
+```
```

Comment:
This method is a ContinuousSampling method, please move to the corresponding section of the docs.

Reply:
Oh, I misunderstood the definition. Thought it was based on the sampling distribution. I see in Meshes the classification is based on the underlying geometry.